src/components/Person/Person.jsx

```diff
@@ -1 +1,58 @@
-// Write code here
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import './Person.scss';
+
+// eslint-disable-next-line
+export const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+// eslint-disable-next-line
+export const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+// eslint-disable-next-line
+export const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
```

Comment:
no need to move it here, you need to pass info form parent component

Reply:
I don't understand. Can you explain?
Why we must save data in App.js. Why not in Person.jsx.
It will be more logical, isn't it?